templates/vue/src/App.vue

```diff
@@ -38,7 +52,26 @@ export default {
       return Object.keys(this.nodes).length === 0
     },
   },
+  watch: {
+    loggedIn(newValue) {
+      if (!newValue) {
+        this.$bvModal.show("loggedOutModal")
+      }
+    },
+  },
   mounted() {
+    if (isLoggedIn()) {
+      var that = this
+      jQuery(function($) {
```

Comment:
Can we avoid using jQuery by any chance? Can we use vanilla JS to check for the event, etc? jQuery will add quite a bit of load to our application.

Reply:
@wynnset Unfortunately, it doesn't seem like we're able to use vanilla JS.
https://wordpress.org/support/topic/heartbeat-api-in-vanilla-js-addeventlistener/
https://stackoverflow.com/questions/54008235/wp-heartbeat-api-in-vanilla-js-addeventlistener-for-custom-events

Reply:
After looking into this and trying a few things, I don't think we have a way of getting around this since WP Heartbeat API is fully relying on jQuery.